src/client/root-reducer.js

```diff
@@ -5,11 +5,13 @@ import lobbyUsers from './lobby/lobby-users-reducer';
 import activeRooms from './lobby/active-rooms-reducer';
 import roomUsers from './rooms/room-users-reducer';
 import roomData from './rooms/room-details-reducer';
+import startGame from './rooms/start-game-reducer';
 
 export default combineReducers({
     lobbyUsers,
     activeRooms,
     roomUsers,
     roomData,
+    startGame,
```

Comment:
I wonder whether there should be a game reducer, user reducer, room reducer? Something to think about